src/mutation/mutation.js

```diff
@@ -105,6 +115,19 @@ export function mutationWithClientMutationId(
       return Promise.resolve(mutateAndGetPayload(input, context, info))
         .then(payload => {
           payload.clientMutationId = input.clientMutationId;
+          if (runAfterMutation) {
+            return Promise.resolve(runAfterMutation(payload))
+              .then(() => payload)
+              .catch(err => {
+                /**
+                 * This shouldn't make the mutation fail. However
+                 * we should let the user know about there error
+                 * & then return the payload
+                 */
+                console.log(err, 'Error in runAfterMutation');
```

Comment:
console.log gives a linter warning, but I think in this case we should at least let the user know that something in there runAfterMutation method failed?